de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/dataaccess/TransactionalJsonProvider.java

```diff
@@ -92,14 +118,30 @@ public void setEd(VirSatTransactionalEditingDomain ed) {
 	protected void preWriteTo(Object object, Class<?> type, Type genericType,
 			Annotation[] annotations, MediaType mediaType,
 			MultivaluedMap<String, Object> httpHeaders, Marshaller marshaller) throws JAXBException {
+		super.preWriteTo(object, type, genericType, annotations, mediaType, httpHeaders, marshaller);
 		marshaller.setEventHandler(eventHandler);
 	}
-
+	
+	@Override
+	public void writeTo(Object object, Class<?> type, Type genericType, Annotation[] annotations, MediaType mediaType,
+			MultivaluedMap<String, Object> httpHeaders, OutputStream entityStream)
+			throws IOException, WebApplicationException {
+		// Useful for debugging
+		super.writeTo(object, type, genericType, annotations, mediaType, httpHeaders, entityStream);
```

Comment:
Do we still need this if it was just for debugging?

Reply:
I think it would be easier to maintain if we leave it there. But I don't have a strong opinion on it, so we could delete them.

Reply:
Okay.